Dockerfile

```diff
@@ -1,7 +1,6 @@
 FROM python:3.11-slim AS base
 RUN apt-get update -qq --fix-missing && apt-get install -y --no-install-recommends \
     ffmpeg \
-    libopus-dev \
```

Comment:
It's working without `libopus` in local. Also, I don't see any places where the program uses it.